spec/classes/rbenv_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
-describe 'rbenv', type: 'class' do
-  let(:facts) { { osfamily: 'Debian' } }
+describe 'rbenv', :type => 'class' do
+  let(:facts) { { :osfamily => 'Debian', :lsbdistcodename => 'xenial' } }
```

Comment:
You'll need to use the `{key: 'value'}` style hash for these changes.

Reply:
sorry i skipped over that by accident fixing now!